src/App.tsx

```diff
@@ -1,51 +1,98 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import {
+  Link, Navigate, NavLink, Route, Routes, useParams,
+} from 'react-router-dom';
+import { FC } from 'react';
+import classNames from 'classnames';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs = [
```

Comment:
It is bad practice to keep all logic in one file, move constants to constants.ts, PageNavLink to components and create pages HomePage, NotFoundPage etc

Reply:
done